### PR TITLE
background image is on body

### PR DESCRIPTION
**Pull request by DumboOctopus**

### CODE REVIEW

**Comment by ghuser1023** [comment]
sounds kinda sus

File: index.html [hunk 1]
```diff
@@ -3,7 +3,7 @@
   <title>what up this is a title</title>
 </head>
 
-  <body>
+  <body style="background-image: url('cake.jpg'); background-repeat: no-repeat;">
```
Comment (ghuser1023) [inline index.html:6]: looks kinda sus

**Review by ghuser1023 — CHANGES_REQUESTED** [review]
I'm making these extremely minor and annoying feature requests that will ruin your day because you were going to play fire emblem but now you gotta fix this

**Review by ghuser1023 — APPROVED** [review]
wait fire emblem isn't even that good

**Comment by DumboOctopus** [comment]
i'm so confused

**Pull request merged**